pkg/kconfig/fuzz.go

```diff
@@ -12,11 +12,6 @@ func FuzzParseKConfig(data []byte) int {
 	return 0
 }
 
-func FuzzParseConfig(data []byte) int {
```

Comment:
Fuzz functions are used by go-fuzz-build.

Reply:
Thanks!
`	_ "github.com/dvyukov/go-fuzz/go-fuzz-build"`
How is it called? Is it better than `go fuzz`?